scalyr_agent/json_lib/parser.py

```
@@ -233,20 +229,22 @@ def parse_value(self):
             elif c == "n":
                 self.__match("null", "unknown identifier")
                 return None
+            elif c is None:
+                if start_pos == 0:
+                    return self.__error("Empty input")
+                else:
+                    return self.__error("Unexpected end-of-text")
+            # this must be after None check because we can not compare with None in python3
             elif c == "-" or "0" <= c <= "9":
                 return self.__parse_number()
             elif c == "}":
                 return self.__error("'}' can only be used to end an object")
-            elif c == "`":
-                return self.__parse_length_prefixed_string()
+            # 2->TODO prefixed string parsing can be removed because json_lib is only used to parse configs.
+            #elif c == "`":
+                #return self.__parse_length_prefixed_string()
```

Let's throw an error in this case.

---

Done.